src/classes/CMT_MetadataAPI_TEST.cls

```diff
@@ -123,10 +123,40 @@ private class CMT_MetadataAPI_TEST {
 
         callback.handleResult(result, context);
 
-        List<Contact> contacts = [SELECT Id, Lastname FROM Contact WHERE Lastname = :ERROR_HANDLER_EXECUTED];
+        List<Contact> contacts = getContacts(ERROR_HANDLER_EXECUTED);
         System.assertEquals(1, contacts.size(), 'A single contact should be created.'); 
     }
 
+    /*************************************************************************************************************
+     * @description Returns accounts matching the provided name
+     **/
+    private static List<Account> getAccounts(String searchName) {
```

Comment:
Nice simple resolution here. No need to call to the query/sosl service given that the unit test is working with only a tiny number of test records.